README.md

```diff
@@ -1 +1,21 @@
-# csdt2024ki49luchakmaks16
\ No newline at end of file
+# Develop
```

Comment:
rename branch from main to develop

Reply:
I wrote to you in a telegram ( @luchmaks )